src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -29,72 +29,66 @@ limitations under the License.
     <attribute side="server" code="0x0000" define="CONTENT_LAUNCHER_ACCEPT_HEADER" type="ARRAY" entryType="CHAR_STRING" length="254" writable="false" optional="false">accept header list</attribute>
     <attribute side="server" code="0x0001" define="CONTENT_LAUNCHER_SUPPORTED_STREAMING_PROTOCOLS" type="BITMAP32" writable="true" optional="false">supported streaming protocols</attribute>
 
-    <command source="client" code="0x00" name="LaunchContent" optional="false">
+    <command source="client" code="0x00" name="LaunchContentRequest" response="LaunchResponse" optional="false">
       <description>Upon receipt, this SHALL launch the specified content with optional search criteria.</description>
       <arg name="autoPlay" type="BOOLEAN"/>
       <arg name="data" type="CHAR_STRING"/>
-      <arg name="search" type="ContentLaunchParamater" array="true"/>
+      <arg name="search" type="Paramater" array="true"/>
     </command>
 
-    <command source="client" code="0x01" name="LaunchURL" optional="false">
+    <command source="client" code="0x01" name="LaunchURLRequest" response="LaunchResponse" optional="false">
       <description>Upon receipt, this SHALL launch content from the specified URL.</description>
       <arg name="contentURL" type="CHAR_STRING"/>
       <arg name="displayString" type="CHAR_STRING"/>
-      <arg name="brandingInformation" type="ContentLaunchBrandingInformation" array="true"/>
+      <arg name="brandingInformation" type="BrandingInformation" array="true"/>
     </command>
 
-    <command source="server" code="0x00" name="LaunchContentResponse" optional="false" disableDefaultResponse="true">
+    <command source="server" code="0x02" name="LaunchResponse" optional="false" disableDefaultResponse="true">
       <description>This command SHALL be generated in response to LaunchContent command.</description>
-      <arg name="contentLaunchStatus" type="ContentLaunchStatus"/>
-      <arg name="data" type="CHAR_STRING"/>
-    </command>
-
-    <command source="server" code="0x01" name="LaunchURLResponse" optional="false" disableDefaultResponse="true">
-      <description>This command SHALL be generated in response to LaunchURL command.</description>
-      <arg name="contentLaunchStatus" type="ContentLaunchStatus"/>
+      <arg name="statusEnum" type="StatusEnum"/>
       <arg name="data" type="CHAR_STRING"/>
     </command>
 
   </cluster>
 
-  <struct name="ContentLaunchAdditionalInfo">
+  <struct name="AdditionalInfo">
     <cluster code="0x050a"/>
     <item name="name" type="CHAR_STRING"/>
     <item name="value" type="CHAR_STRING"/>
   </struct>
 
-  <enum name="ContentLaunchMetricType" type="ENUM8">
+  <enum name="MetricTypeEnum" type="ENUM8">
```

Comment:
This type is called "MetricType" in the spec.

Reply:
Created spec issue: https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/4801. Can you please help with tags and assignee?